packages/services/src/services/Locale/Locale.js

```diff
@@ -6,7 +6,7 @@ import { ipcinfoCookie, geolocation } from '@carbon/ibmdotcom-utilities';
  * @constant {string | string} Host for the Locale API call
  * @private
  */
-const _host = process.env.TRANSLATION_HOST || 'https://www.ibm.com';
+const _host = process.env.TRANSLATION_HOST || 'https://1.www.s81c.com';
```

Comment:
The `TranslationAPI` also uses the `TRANSLATION_HOST`, I don't think this affects it since it's pointing to what looks like an akamai host, but wanted to make sure?

Reply:
Doublechecked, it seems to work:
https://1.www.s81c.com/common/v18/js/data/jsononly/usen.json